### PR TITLE
Update Keyring Backends with `celestia-node` PR #1565

### PR DESCRIPTION
**Pull request by jcstein**
This resolves issue with unsupported keyring backends in our documentation. It specifies that `test` and `file` are available, but others are not.

## Overview

Resolves https://github.com/celestiaorg/docs/issues/413

- [x] Remove `./cel-key --help` menu, because it shows unsupported keyring backends https://github.com/celestiaorg/docs/pull/414/commits/b199f319ff7c04ba4a3151402ca87a3403660839
- [x] add note about supported keyring backends in `docs/developers/celestia-node-key.mdx` https://github.com/celestiaorg/docs/pull/414/commits/a9d1d7c90b7c85d5ac50447d18528767d2987294
- [x] change `--node.type <node_type> --p2p.network <network>` to `--keyring-dir ~/.celestia-<node_type>-<p2p_network>/keys`

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203749804732586

### CODE REVIEW

File: docs/developers/celestia-node-key.mdx [hunk 1]
```diff
@@ -48,6 +48,15 @@ make install-key
 
 For the purpose of this guide, we will use the `make cel-key` command.
 
+### `keyring-backend` usage
+
+`keyring-backend` configures the keyring's backend, where the keys are stored.
+
+Supported options are: `file` and `test`. The default used in Celestia's documentation is `test`.
```
Comment (renaynay) [inline docs/developers/celestia-node-key.mdx:55]:
```suggestion
Fully supported options are: `file` and `test`. Other options are also available to use at user's discretion. The default used in celestia-node and Celestia's documentation is `test`.
```

File: docs/developers/celestia-node-key.mdx [hunk 2]
```diff
@@ -48,6 +48,15 @@ make install-key
 
 For the purpose of this guide, we will use the `make cel-key` command.
 
+### `keyring-backend` usage
+
+`keyring-backend` configures the keyring's backend, where the keys are stored.
```
Comment (renaynay) [inline docs/developers/celestia-node-key.mdx:53]:
```suggestion
`keyring-backend` configures the keyring's backend, how the keys are stored.
```

**Comment by renaynay** [comment]
@jcstein 
Also something that should be included in the PR is removing `--node.type` and `--p2p.network` flags from cel-key commands and just passing `-keyring-dir` with default directory paths

**Mocha**: `~/.celestia-<node_type>-mocha/keys`
**Arabica**: `~/.celestia-<node_type>-arabica-2/keys`

**Comment by jcstein** [comment]
thanks @renaynay, I made suggested changes https://github.com/celestiaorg/docs/pull/414/commits/a4841fb854bd460d3625ed550741c9c551046070

I left in the instructions for user to still have option to use `--node.type` and `--p2p.network`. Let me know if those no longer work and will make change

**Pull request closed without merging**